Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp

```diff
@@ -492,6 +492,11 @@ bool property_accepts_value(PropertyID property_id, StyleValue& style_value)
                             property_generator.append(R"~~~(
         if (style_value.is_image())
             return true;
+)~~~");
+                        } else if (type_name == "linear-gradient") {
```

Comment:
Is this needed? I haven't looked into gradients that much, but from what I can tell a gradient is part of the `<image>` type so should be valid everywhere that images are, without special handling.

Instead, I think the `image` branch above should just check for `style_value.is_linear_gradient()` too. Do correct me if I'm wrong though. :^)

Reply:
I think you're right :)